vulture_allowlist.py

```diff
@@ -82,6 +82,8 @@
 Edge.meta
 Monkey.guid
 Monkey.launch_time
+Monkey.dead
+Monkey.ttl_ref
```

Comment:
What still uses these?

Reply:
`resources/monkey.py` accesses them as dictionary keys.

Reply:
Which is needed during agent wakeup to register an agent with the Island? Can't we get rid of that now?

Reply:
There's work for that planned next week.